pawn.json

```diff
@@ -5,6 +5,7 @@
     "output": "test/maptest.amx",
     "include_path": "include",
     "dependencies": ["sampctl/pawn-stdlib", "BigETI/pawn-memory:2.0.1"],
+    "dev_dependencies": ["sampctl/samp-stdlib"],
```

Comment:
Why would like to know why sampctl/samp-stdlib would be a dev dependency.

Reply:
Because: I believe, everyone who contributes, tests the results before pushing. If someone clonse your repo and makes changes to library, then he/she'd like to run [tests](https://github.com/BigETI/pawn-map/blob/master/test/maptest.pwn) to see the build output. So that, *sampctl package ensure* is not enough, because it just gets *pawn-stdlib* and *pawn-memory*, but you have *<a_samp>* included in it.

Reply:
When someone **sampctl package install**s your _pawn-map_ package, sampctl will only ensure "dependencies" of it, it won't touch "dev_dependencies".

Reply:
You probably want to include certain libraries from "pawn-stdlib" instead of having it fully included by *a_samp*. That's another thing.